docs/tutorial-attachments.rst

```diff
@@ -144,3 +144,15 @@ About compression
 The server does not compress the files.
 
 We plan to enable compression at the HTTP level (`Bug 1339114 <https://bugzilla.mozilla.org/show_bug.cgi?id=1339114>`_) for when clients fetch the attachment using the ``Accept-Encoding: gzip`` request header.
+
+In the admin tool
+-----------------
+
+The Remote Settings administration tool supports attachments as well. If a collection has a record schema and attachments are "enabled" for that collection, then editors will be able to upload attachments as part of editing records.
+
+The controls for attachments in a given collection are in the ``attachment`` field in the collection data. ``attachment`` should be an object and it can have the following properties:
+
+- ``enabled``: boolean, true to enable attachments for this collection
+- ``required``: boolean, true if records in this collection must have an attachment
+
+In stage, you can also enable attachments on a collection on the edit screen by checking the "File attachment" section.
```

Comment:
The YML files are ran both in stage and prod, and changes in the UI will be overridden at the next run.

Also I realize we have this bug around: https://github.com/Kinto/kinto-admin/issues/616

Reply:
What is the status of self-service in stage? I guess I was still operating in a world where people created their own collections in stage, and thus they wouldn't be changed by the YAML files. In particular, the collection I was asked about (personality-provider-recipe) doesn't exist in the YAML file at https://github.com/mozilla-services/remote-settings-permissions/blob/master/kinto.stage.yaml.

Reply:
Yes, it does: https://github.com/mozilla-services/remote-settings-permissions/blob/master/kinto.prod.yaml#L499
The PROD collection are all applied to STAGE.

If you create a collection that is not the YAML file, then yes indeed, it won't be overwritten by the YAML (nor deleted)